staging/src/k8s.io/apiserver/pkg/apis/config/v1/types.go

```diff
@@ -87,4 +89,7 @@ type KMSConfiguration struct {
 	// Timeout for gRPC calls to kms-plugin (ex. 5s). The default is 3 seconds.
 	// +optional
 	Timeout *metav1.Duration `json:"timeout,omitempty"`
+	// Probe when set to true will instruct KMS Provider to setup a healthz check against KMS-Plugin.
+	// +optional
+	Probe bool `json:"probe,omitempty"`
```

Comment:
if this is controlling whether we connect kms health to API server healthz, can we name this something with `healthz` in the name? If something like https://github.com/kubernetes/kubernetes/pull/78458 progresses, we'll likely want distinct control over connecting kms health to API server healthz and readyz

Reply:
Renamed to InstallHealthz.